### DISCUSSION TITLE
Fbs Pro License for poor college student

### DISCUSSION THREAD

**Discussion by Zeth3n** [AnthonyMichaelTDM/DRG-Save-Editor#15] [General]
Hey Anthony,

Appreciate the work you've put into keeping this going after original author couldn't continue it. Saw your note about not having the license to package the zip file and wanted to help out a bit by getting you a license. Was straightforward for me but that's because I do it day in and day out for work, for most people this should make the work you're doing much for accessible. Might also try to pitch in on the cosmetic GUID hunt if I can find the time.

Just shoot an email to jesustest52@gmail.com (one of many throwaway emails) because github sadly doesn't have a PM system so I'll reply back to you with the link. Apparently the download link expires in 10 days with no way to get a new one so hopefully I can get it to you within that time frame.

Thanks again for all your work!

**Comment by AnthonyMichaelTDM** [comment]
One thing you could do to emulate a private message is create a private repository and add me as a collaborator

**Comment by Zeth3n** [comment]
Done, good plan.
